Plash/OpenURLView.swift

```diff
@@ -10,7 +10,7 @@ struct OpenURLView: View {
 			return ""
 		}
 
-		return url.absoluteString
+		return url.absoluteString.removingPercentEncoding ?? url.absoluteString
```

Comment:
I did this to prevent the escape characters for the brackets from showing when we're presenting a human-readable URL. Is this too invasive?

Reply:
Why would it be too invasive?

Reply:
Fair enough.

Reply:
I was genuinely asking why you think it would be too invasive? Because that was not clear to me.

Reply:
Ah, I see. I was just concerned about potential side effects since I didn't know at the time if you rely on the URL string being (un)escaped elsewhere. It looks like I was overthinking it.